front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -255,8 +254,6 @@ class SettingsCustomizeTab extends PureComponent<
     const { locale, tenant } = this.state;
 
     if (!isNilOrError(locale) && !isNilOrError(tenant)) {
-      const homepageInfoPage = tenant.data.attributes.homepage_info_multiloc;
```

Comment:
Not reviewing yet, but `homepage_info_multiloc?: Multiloc;` on IAttributesDiff also needs to be deleted.

Reply:
Just a preliminary review of the folder structure and decisions made so far would be cool so I don't do too much work that might need to be reverted